airflow/bin/cli.py

```diff
@@ -1082,15 +1086,17 @@ def flower(args):
         )
 
         with ctx:
-            os.execvp("flower", ['flower', '-b', broka, address, port, api, flower_conf])
+            os.execvp("flower", ['flower', '-b', broka, address, port, api, flower_conf,
+                                 url_prefix])
 
         stdout.close()
         stderr.close()
     else:
         signal.signal(signal.SIGINT, sigint_handler)
         signal.signal(signal.SIGTERM, sigint_handler)
 
-        os.execvp("flower", ['flower', '-b', broka, address, port, api, flower_conf])
+        os.execvp("flower", ['flower', '-b', broka, address, port, api, flower_conf,
+                             url_prefix])
```

Comment:
I'm not too familiar with this bit of the code. Can you confirm that this url_prefix here maps to FLOWER_URL_PREFIX, and not to WEB_SERVER_URL_PREFIX?

Reply:
Yes, it comes from the line 1611 (`func': flower`). I confirm this works (I use `/airflow/` in my URL for web service, and `/airflow/flower` for flower UI, so I have `/airflow/admin` for the UI, `/airflow/api` for the experimental REST api, `/airflow/flower` for flower ui)